packages/popover/src/vaadin-popover.js

```diff
@@ -578,27 +577,32 @@ class Popover extends PopoverPositionMixin(
   }
 
   /** @private */
-  __openedOrTargetChanged(opened, target) {
-    if (target) {
-      target.setAttribute('aria-expanded', opened ? 'true' : 'false');
-
-      if (opened) {
-        target.setAttribute('aria-controls', this.__overlayId);
-      } else {
-        target.removeAttribute('aria-controls');
-      }
-    }
+  __computeEffectiveTarget(target, ariaTarget) {
```

Comment:
It looks like this can be removed again.

Reply:
Thanks, I missed it. Done.